### PR TITLE
Replaced speeds relative to constant

### LINKED ISSUE #133: Switch speeds to be relative to MAX_DRIVE_POWER constant
Instead of having drive speeds just written throughout the code make them relative to the MAX_DRIVE_POWER constant in core/constants.py

### PR DESCRIPTION
**Pull request by Viktor-Butkovich**
closes #133
Replaced hardcoded speed values with ones relative to core.MAX_DRIVE_POWER

### CODE REVIEW

File: algorithms/gps_navigate.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def get_approach_status(goal, location, start, tolerance=core.WAYPOINT_DISTANCE_
     return core.ApproachState.APPROACHING
 
 
-def calculate_move(goal, location, start, speed=150):
+def calculate_move(goal, location, start, speed=0.6*core.MAX_DRIVE_POWER):
```
Comment (Byrdman32) [inline algorithms/gps_navigate.py]: Make sure that this file has:
`#include core.constants`

File: algorithms/gps_navigate.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ def calculate_move(goal, location, start, speed=150):
     logger.debug(f"Target distance: {target_distance}")
 
     if target_distance < 0.01:
-        speed = 100
+        speed = 0.4*core.MAX_DRIVE_POWER
```
Comment (Byrdman32) [inline algorithms/gps_navigate.py]: Make sure that this file has:
`#include core.constants`

File: core/states/approaching_gate.py [hunk 3]
```diff
@@ -151,7 +151,7 @@ async def run(self) -> RoverState:
                             core.Coordinate(point[0], point[1]),
                             interfaces.nav_board.location(),
                             start,
-                            250,
+                            core.MAX_DRIVE_POWER,
```
Comment (Byrdman32) [inline core/states/approaching_gate.py:177]: Make sure that this file has:
`#include core.constants`

File: core/states/approaching_marker.py [hunk 4]
```diff
@@ -64,7 +64,7 @@ async def run(self) -> RoverState:
             distance = tags[0].distance
             angle = tags[0].angle
 
-            left, right = algorithms.follow_marker.drive_to_marker(100, angle)
+            left, right = algorithms.follow_marker.drive_to_marker(0.4*core.MAX_DRIVE_POWER, angle)
```
Comment (Byrdman32) [inline core/states/approaching_marker.py:87]: Make sure that this file has:
`#include core.constants`

File: core/states/avoidance.py [hunk 5]
```diff
@@ -90,7 +90,7 @@ async def run(self) -> RoverState:
                         core.Coordinate(new_lat, new_lon),
                         interfaces.nav_board.location(),
                         previous_loc,
-                        250,
+                        core.MAX_DRIVE_POWER,
```
Comment (Byrdman32) [inline core/states/avoidance.py:111]: Make sure that this file has:
`#include core.constants`

File: example/obstacle_avoidance_example.py [hunk 6]
```diff
@@ -46,7 +46,7 @@ def main() -> None:
             == core.ApproachState.APPROACHING
         ):
             left, right = algorithms.gps_navigate.calculate_move(
-                core.Coordinate(new_lat, new_lon), interfaces.nav_board.location(), previous_loc, 250
+                core.Coordinate(new_lat, new_lon), interfaces.nav_board.location(), previous_loc, core.MAX_DRIVE_POWER
```
Comment (Byrdman32) [inline example/obstacle_avoidance_example.py:56]: Make sure that this file has:
`#include core.constants`

**Review by Byrdman32 — APPROVED** [review]

**Review by jpvf2d — APPROVED** [review]

**Pull request merged**